### PR TITLE
Feat(Input tokens): Bump input border colors for increased contrast

### PR DESCRIPTION
**Pull request by RostiMelk**
- Bumped border color for input fields from 800 -> 700 (dark scheme).
- Added background color to disabled inputs to better indicate that they are in fact disabled.

### CODE REVIEW

File: src/theme/defaults/colorTokens.ts [hunk 1]
```diff
@@ -443,6 +443,7 @@ export const defaultColorTokens: ThemeColorTokens = {
         fg: ['800', '200'],
       },
       'disabled': {
+        bg: ['50', '950'],
```
Comment (RostiMelk) [inline src/theme/defaults/colorTokens.ts:446]: Added background to better indicate a disabled input

File: src/theme/defaults/colorTokens.ts [hunk 2]
```diff
@@ -427,7 +427,7 @@ export const defaultColorTokens: ThemeColorTokens = {
       '*': {
         _blend: ['multiply', 'screen'],
         bg: ['white', 'black'],
-        border: ['200', '800'],
+        border: ['300', '700'],
```
Comment (RostiMelk) [inline src/theme/defaults/colorTokens.ts:430]: Increased constrast for both schemes

**Review by joneidejohnsen — DISMISSED** [review]

**Review by mariuslundgard — DISMISSED** [review]
Looks great, thanks for doing this 🙇

File: src/theme/defaults/colorTokens.ts (hunk 2 above)
Comment (mariuslundgard) [reply src/theme/defaults/colorTokens.ts:430]: We talked offline, but want to keep `200` for the light mode for now 🙏

**Review by mariuslundgard — APPROVED** [review]
LGTM 👍

**Pull request merged**